docs/libraries/messaging.md

```
@@ -1,775 +1,825 @@
+
+```csharp
+// src/Shared/Dhadgar.Contracts/YourDomain/Events.cs
+namespace Dhadgar.Contracts.YourDomain;
+
+public record SomethingHappened(
+    Guid EntityId,
+    string Details,
+    DateTimeOffset OccurredAtUtc);
+```
+
+2. **Create the consumer class** in your service:
```

markdownlint-cli2 (0.20.0)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)